docs/shared/recipes/plugins/create-install-package.md

```diff
@@ -0,0 +1,159 @@
+# Creating an Install Package
+
+{% youtube
+src="https://www.youtube.com/embed/ocllb5KEXZk"
+title="Build your own CLI"
+width="100%" /%}
+
+A [Nx Plugin Preset](/extending-nx/recipes/create-preset) allows you to take control over the shape of an Nx workspace setup, so you can customize it to best fit your situation. This is possible because a preset can be passed directly to the `create-nx-workspace` command:
+
+```shell
+npx create-nx-workspace --preset my-plugin
+```
+
+You might however want to create your own `create-{x}` package, whether that's for marketing purposes or easier discovery. When a package named create-{x} is published to npm, it can be invoked via any of the following:
+
+- npx create-{x}
+- yarn create {x}
+- npm init {x}
+- pnpm init {x}
+
+These packages are used to set up a new project in some form. As an example, consider `create-nx-plugin`, `create-nx-workspace`, or perhaps `create-react-app`.
+
+## Generating a "Create Package"
```

Comment:
You need some transition text before this section explaining how Nx helps you to create packages that work with npm's `create-*` feature and the rest of the article explains how to use it.